test/fit_tests/deploy/os_ova_install.py

```diff
@@ -4,7 +4,11 @@
 Author(s):
 George Paulos
 
-This script installs blank OVA template.
+This script installs blank OVA templates into selected stack.
+if numvms is greater than 1, MAC address installation on OVA is done by stack number convention
```

Comment:
I think that a better plan is to have a hex counter rather than tying the MAC to a stack number. What if the stack label is not a number? A counter would be more deterministic and transparent. In our MAC scheme we put in a full byte of space between each stack address for additional OVAs just for this situation. Assume that the base MAC always ends with 00 then count up from there based upon the supplied MAC from stack config.